apps/web/src/pages/ApplicantDashboardPage/components/FunctionalCommunityListItem.tsx

```diff
@@ -0,0 +1,196 @@
+      })}
+    </span>
+  );
+  const missingWork = (
+    <span>
+      <ExclamationTriangleIcon
+        data-h2-color="base(error)"
+        {...sharedIconStyling}
+      ></ExclamationTriangleIcon>
+      {intl.formatMessage({
+        defaultMessage: "<red>Missing work info</red>",
+        id: "ZrLeP8",
+        description:
+          "Phrase marking incomplete community work opportunities interest",
+      })}
+    </span>
+  );
+
+  const generateMetaDataJobInterest = (
+    jobInterest: boolean | null | undefined,
+  ): JSX.Element => {
+    if (jobInterest === null || jobInterest === undefined) {
+      return missingWork;
+    }
+    return jobInterest ? interestedWork : notInterestedWork;
+  };
+
+  const interestedTraining = (
+    <span>
+      <PresentationChartLineIcon
+        data-h2-color="base(success)"
+        {...sharedIconStyling}
+      ></PresentationChartLineIcon>
+      {intl.formatMessage({
+        defaultMessage: "Interested in training",
+        id: "ERsZAD",
+        description:
+          "Phrase marking interest in community training opportunities",
+      })}
+    </span>
+  );
+  const notInterestedTraining = (
+    <span>
+      <XCircleIcon
+        data-h2-color="base(gray.lighter)"
+        {...sharedIconStyling}
+      ></XCircleIcon>
+      {intl.formatMessage({
+        defaultMessage: "Not interested in training",
+        id: "8wU0cq",
+        description:
+          "Phrase marking lack of interest in community training opportunities",
+      })}
+    </span>
+  );
+  const missingTraining = (
+    <span>
+      <ExclamationTriangleIcon
+        data-h2-color="base(error)"
+        {...sharedIconStyling}
+      ></ExclamationTriangleIcon>
+      {intl.formatMessage({
+        defaultMessage: "<red>Missing training info</red>",
+        id: "bupAcV",
+        description:
+          "Phrase marking incomplete community training opportunities interest",
+      })}
+    </span>
+  );
+
+  const generateMetaDataTrainingInterest = (
+    trainingInterest: boolean | null | undefined,
+  ): JSX.Element => {
+    if (trainingInterest === null || trainingInterest === undefined) {
+      return missingTraining;
+    }
+    return trainingInterest ? interestedTraining : notInterestedTraining;
+  };
+
+  type MetaDataProps = React.ComponentProps<
+    typeof PreviewList.Item
+  >["metaData"];
+  type MetaDataPropItem = MetaDataProps[number];
+  const metaDataProps: MetaDataPropItem[] = [
+    {
+      key: "job-interest",
+      type: "text",
+      children: generateMetaDataJobInterest(
+        functionalCommunityListItemFragment.jobInterest,
+      ),
+    },
+    {
+      key: "training-interest",
+      type: "text",
+      children: generateMetaDataTrainingInterest(
+        functionalCommunityListItemFragment.trainingInterest,
+      ),
+    },
+  ];
+
+  return (
+    <>
+      <PreviewList.Item
+        title={
+          functionalCommunityListItemFragment?.community?.name?.localized ?? ""
```

Comment:
We should have some text as a fallback instead of an empty string here since it is a title

Reply:
True, done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/12607/commits/683a9b6dc5c61d485f258e9a64f7abb96c647fe5